lms/templates/themable-footer.html

```diff
@@ -0,0 +1,13 @@
+<%!
```

Comment:
See [thread above about this file's name](https://github.com/edx/edx-platform/pull/8271/files#r37611297).